src/gatekeeper/Dockerfile_x86_64

```diff
@@ -0,0 +1,40 @@
+################################################################################
```

Comment:
This is the right thing to do - the only thing missing here is a delete or rename of the original Dockerfile to Dockerfile_x86_64. (If I check out this branch both  `src/gatekeeper/Dockerfile_x86_64` and `src/gatekeeper/Dockerfile` exist.  src/gatekeeper/Dockerfile should be deleted now we've split this by platform.)

Reply:
(I'm fixing this in a follow up PR to enable us to merge the ppc64le changes.)